_Sidebar.md

```diff
@@ -1,11 +1,11 @@
 **TypeScript Language Basics**
 
-* [[Basic Types]]
-* [[Interfaces]]
-* [[Classes]]
-* [[Modules]]
-* [[Functions]]
-* [[Generics]]
+* [Basic Types](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Basic%20Types.md)
+* [Interfaces](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Interfaces.md)
+* [Classes](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Classes.md)
+* [Namespaces and Modules](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Namespaces%20and%20Modules.md)
+* [Functions](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Functions.md)
+* [Generics](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Generics.md)
 * [[Common Errors]]
 * [[Integrating with Build Tools]]
 * [[Compiler Options]]
```

Comment:
I would say everything related to the language/toolset goes to the handbook. things related to contributing, releases, compiler API stays in the wiki